migrations/V152__Update_Possible_Operational_Solution_Contacts.sql

```diff
@@ -0,0 +1,32 @@
+-- Remove Hung Van and Yolanda Villanova as Solution Contacts

```

Comment:
Can you update this please? This is the comment from the source migration.

Reply:
Good catch. Done. 👍